test/runner.js

```diff
@@ -87,10 +99,42 @@ if ( program.nodeTotal > 1 ) {
 
 files.forEach( setup.addFile );
 
-mocha.addFile( path.join( __dirname, 'load-suite.js' ) );
-
-mocha.run( function( failures ) {
-	process.on( 'exit', function() {
-		process.exit( failures ); //eslint-disable-line no-process-exit
+if ( program.watch ) {
+	console.log( chalk.green( 'Watch mode enabled' ) );
+
+	let runAgain = false;
+	let runner = null;
+
+	let rerun = () => {
+		console.log( chalk.green( 'Running tests at: ' ),
+									chalk.yellow( new Date( ) ) );
```

Comment:
>I'm pretty sure I was getting eslint warnings without the space. I'll double check

Spaces are necessary when arguments are present, but can be left out otherwise.

- [Guideline](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#arrays-and-function-calls)
- [ESLint rule](http://eslint.org/docs/rules/space-in-parens) (we configure to "always")